sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,519 @@
+      END
+    ) AS customize_home_pocket_users,
+    COALESCE(SUM(customize_home_pocket), 0) AS customize_home_pocket,
+    --customize_home_recently_saved_users
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN customize_home_recently_saved > 0
+          THEN client_id
+      END
+    ) AS customize_home_recently_saved_users,
+    COALESCE(SUM(customize_home_recently_saved), 0) AS customize_home_recently_saved,
+    --customize_home_recently_visited_users
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN customize_home_recently_visited > 0
+          THEN client_id
+      END
+    ) AS customize_home_recently_visited_users,
+    COALESCE(SUM(customize_home_recently_visited), 0) AS customize_home_recently_visited,
+  FROM
+    product_features
+  WHERE
+    submission_date >= start_date
+  GROUP BY
+    1
+)
+SELECT
+  d.submission_date,
+  dau
+/*logins*/
+  ,
+  logins_deleted_users,
+  logins_deleted,
+  logins_modified_users,
+  logins_modified,
+  currently_stored_logins_users,
+  currently_stored_logins
+/*credit card*/
+  ,
+  credit_cards_deleted_users,
+  credit_cards_deleted,
+  currently_stored_credit_cards_users,
+  currently_stored_credit_cards
+/*addresses*/
+  ,
+  addresses_deleted_users,
+  addresses_deleted,
+  addresses_modified_users,
+  addresses_modified,
+  currently_stored_addresses_users,
+  currently_stored_addresses
+/*bookmark*/
+  ,
+  bookmarks_add_users,
+  bookmarks_add,
+  bookmarks_delete_users,
+  bookmarks_delete,
+  bookmarks_edit_users,
+  bookmarks_edit,
+  bookmarks_open_users,
+  bookmarks_open,
+  metrics_desktop_bookmarks_count_users,
+  metrics_desktop_bookmarks_count,
+  metrics_mobile_bookmarks_count_users,
+  metrics_mobile_bookmarks_count,
+  metrics_has_desktop_bookmarks_users,
+  metrics_has_desktop_bookmarks,
+  metrics_has_mobile_bookmarks_users,
+  metrics_has_mobile_bookmarks
+/*privacy*/
+  ,
+  etp_standard_users,
+  etp_standard,
+  etp_strict_users,
+  etp_strict,
+  etp_custom_users,
+  etp_custom,
+  etp_disabled_users,
+  etp_disabled
+/*Tab count*/
+  ,
+  metrics_private_tabs_open_count_users,
+  metrics_private_tabs_open_count,
+  metrics_tabs_open_count_users,
+  metrics_tabs_open_count
+/*Default browser*/
+  ,
+  metrics_default_browser_users,
+  metrics_default_browser
+/*Notification*/
+  ,
+  metrics_notifications_allowed_users,
+  metrics_notifications_allowed,
+  events_marketing_notification_allowed_users,
+  events_marketing_notification_allowed
+/*Customize Home*/
+  ,
+  customize_home_contile_users,
+  customize_home_contile,
+  customize_home_jump_back_in_users,
+  customize_home_jump_back_in,
+  customize_home_most_visited_sites_users,
+  customize_home_most_visited_sites,
+  customize_home_pocket_users,
+  customize_home_pocket,
+  customize_home_recently_saved_users,
+  customize_home_recently_saved,
+  customize_home_recently_visited_users,
+  customize_home_recently_visited
+FROM
+  dau_segments d
+LEFT JOIN
+  product_features_agg p
+ON
```

Comment:
For consistency with the other query files and nicer syntax, we should consider using `USING()` here instead of `ON`